runtime/node.go

```diff
@@ -51,5 +59,35 @@ func (conf nodeJsRuntime) GetYamlTemplate() string {
 # Note that package root will correspond to filesystem root (/) in OSv image.
 # Example value: /server.js
 main: <filepath>
+
+# OPTIONAL
+# A list of Node.js args.
+# Example value: node_args:
+#                   - --require module1
+node_args:
+   - <list>
+
+# OPTIONAL
+# A list of command line args used by the application.
+# Example value: args:
+#                   - argument1
+#                   - argument2
+args:
+   - <list>
 ` + conf.CommonRuntime.GetYamlTemplate()
 }
+
+//
+// Utility
+//
+
+func (conf nodeJsRuntime) concatNodeArgs() string {
+	if len(conf.NodeArgs) > 0 {
+		return strings.Join(conf.NodeArgs, " ")
+	} else {
+		// This is a workaround since runscript is currently unable to
+		// handle empty environment variable as a parameter. So we set
+		// dummy value unless user provided some actual value.
+		return "--no-deprecation"
```

Comment:
I've got a bit of a problem with `--no-deprecation`. Do you perhaps know what is the default?

What if we simply used `--` instead? From man page: `--     Indicate the end of node options. Pass the rest of the arguments to the script.`